cocinaborcelle/src/pages/api/restaurante/[slug].ts

```diff
@@ -0,0 +1,45 @@
+import { NextApiRequest, NextApiResponse } from "next"
+import { PrismaClient } from "@prisma/client" 
+
+const prisma = new PrismaClient()
```

Comment:
```suggestion
import prisma from "../../../../prisma/index"

```

em vez de criar um novo cliente prisma, em prisma/index já existe uma instância criada para ser usada globalmente em toda aplicação. Isso evitar criar múltiplas instâncias de cliente prisma, evitando vazamento de memória na aplicação